backend/pkg/api/routes.go

```diff
@@ -235,6 +235,12 @@ func (api *API) routes() *chi.Mux {
 				r.Post("/users", api.handleCreateUser())
 				r.Delete("/users/{principalID}", api.handleDeleteUser())
 
+				// Wasm Transforms
+				r.Get("/transforms", api.handleListTransforms())
+				r.Get("/transforms/{transformID}", api.handleGetTransform())
+				r.Put("/transforms", api.handleDeployTransform())
+				r.Delete("/transforms/{transformID}", api.handleDeleteTransform())
+
```

Comment:
I could be missing some context, but do we actually need this? This unfortunately might be a case of looking at the existing codebase and doing extra work that may not be necessary.
The connect gateway package already exposes HTTP REST API based on `google.api.http` annotations in the proto files. So unless we specifically want these paths and implement these handlers, I think this is unnecessary? @weeco to chime in.

Reply:
I'm fine either way. I was honestly confused when I saw the connect stuff in the standard API because as you say, it should already exist. But I didn't want to not do it and need it 😅

Reply:
Yeah this is obsolete, it will happen automatically through grpc-gateway.

Reply:
I think we can probably delete these routes and `backend/pkg/api/handle_transforms.go`.